pkg/internal/checkup/trex/client.go

```diff
@@ -130,7 +130,7 @@ func (t Client) getTrexServiceJournalctl(vmiName string) (string, error) {
 }
 
 func (t Client) runTrexConsoleCmd(vmiName, command string) (string, error) {
-	shellCommand := fmt.Sprintf("/bin/sh -c \"cd /opt/trex && echo %q | ./trex-console -q\"", command)
+	shellCommand := fmt.Sprintf("cd /opt/trex && echo %q | ./trex-console -q", command)
```

Comment:
Commit message: The command was executed in a sub-shell [1].
The advantage of it was that it did not change the current shell.
The disadvantage is that it is a bit slower.

[1] https://tldp.org/LDP/abs/html/subshells.html#:~:text=A%20subshell%20is%20a%20separate,process%20a%20list%20of%20commands.

Reply:
DONE